packages/fluentui/react-northstar/src/components/Carousel/CarouselNavigation.tsx

```diff
@@ -116,13 +120,16 @@ export const CarouselNavigation: ComponentWithAs<'ul', CarouselNavigationProps>
     rtl: context.rtl,
   });
 
-  const handleItemOverrides = variables => predefinedProps => ({
-    onClick: (e, itemProps) => {
-      _.invoke(props, 'onItemClick', e, itemProps);
-      _.invoke(predefinedProps, 'onClick', e, itemProps);
-    },
-    variables: mergeVariablesOverrides(variables, predefinedProps.variables),
-  });
+  const handleItemOverrides = (variables: any, disableClickableNav: boolean) => predefinedProps =>
+    disableClickableNav
```

Comment:
I think you should conditionally replace the `onClick` prop and not all props. Currently by setting `disableClickableNav` `variables` are no longer passed down

Reply:
```js
onClick: !disableClickableNav && (e, itemProps) => {...}
```

Reply:
moved this variable to carousel.tsx, thanks